common/sgx/verifier.c

```diff
@@ -390,6 +394,22 @@ static oe_result_t _fill_with_known_claims(
 
     if (format_type != SGX_FORMAT_TYPE_LOCAL)
     {
+        // TCB status.
+        OE_CHECK(oe_sgx_add_claim(
+            &claims[claims_index++],
+            OE_CLAIM_TCB_STATUS,
+            sizeof(OE_CLAIM_TCB_STATUS),
+            tcb_status,
+            strlen(tcb_status) + 1));
```

Comment:
As in my previous comment, I don't understand why these are strings instead of strongly typed values like in an enum like we have for oe_result_t.   Strings are bug prone on the app side if the app will programmatically check for specific values, and are only useful for display if the user speaks English.

Reply:
Updated to enum type.